canonical/juju-dashboard
# WD-5220 - Add audit logs filter panel

## Done

- Add a panel for filtering audit logs (this doesn't refetch from the API yet as it needs a couple of other PRs to land first).

## QA

- Connect your dashboard to the staging JIMM.
- Go to the Logs page.
- Click the filter button.
- Click on some of the fields and depending on what data you have it might display suggestions.
- Fill out some fields and click the Filter button.
- You should see the currently applied filter chips.
- Dismiss some chips and the filters should change.
- Connect your dashboard to the production JIMM (the default JAAS websocket).
- Go to a model and click the Logs tab and then the Audit logs toggle.
- You should be able to open the filter panel from the new action bar.

## Details

https://warthogs.atlassian.net/browse/WD-5220
